node/setup.go

```diff
@@ -228,13 +229,16 @@ func createPeerManager(
 	options := p2p.PeerManagerOptions{
 		SelfAddress:            selfAddr,
 		MaxConnected:           maxConns,
+		MaxOutgoingConnections: maxConns / 2,
```

Comment:
We should add this to the configuration file instead of just selecting 1/2.

Reply:
yeah I agree

Reply:
ok